src/Protocol/DFRN.php

```diff
@@ -2923,6 +2928,9 @@ public static function autoRedir(App $a, $contact_nick)
 			$baseurl = substr($baseurl, $domain_st + 3);
 			$nurl = Strings::normaliseLink($baseurl);
 
+			$r = DBA::selectFirst("user", ["uid"], ["nickname" => DBA::escape($contact_nick)], []);
```

Comment:
Please use `Friendica\Model\User::getByNickname($contact_nick)` instead. Are you using an IDE to check for namespaces?

Reply:
New commit changes to the API call.  I'm not using an IDE.